internal/provider/resource_local_file.go

```diff
@@ -232,8 +260,15 @@ func (n *localFileResource) Read(ctx context.Context, req resource.ReadRequest,
 		return
 	}
 
-	outputChecksum := sha1.Sum(outputContent)
-	if hex.EncodeToString(outputChecksum[:]) != state.ID.ValueString() {
+	checksums := genFileChecksums(outputContent)
+	state.ContentMd5 = types.StringValue(checksums.md5Hex)
+	state.ContentSha1 = types.StringValue(checksums.sha1Hex)
+	state.ContentSha256 = types.StringValue(checksums.sha256Hex)
+	state.ContentBase64sha256 = types.StringValue(checksums.sha256Base64)
+	state.ContentSha512 = types.StringValue(checksums.sha512Hex)
+	state.ContentBase64sha512 = types.StringValue(checksums.sha512Base64)
```

Comment:
Do we need set these values? It doesn't look like we are using them further down in the `read()` method. I think you can just copy and paste the implementation you did for the `read()` method in `resource_local_sensitive_file` which looks correct to me. The implementation for the CRUD methods should mostly be the same for both `resource_local_file `and `resource_local_sensitive_file` since the only difference is marking some attributes as `sensitive` in the schema